<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LockinAmplifierControl</class>
 <widget class="QWidget" name="LockinAmplifierControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>241</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Lock-in Amplifier Control</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="GBInput">
       <property name="title">
        <string>Signal Input</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="LRange">
            <property name="text">
             <string>Sensitivity</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="SBRange">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>0.001000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="BAutoRange">
            <property name="text">
             <string>Auto Range</string>
            </property>
            <property name="icon">
             <iconset resource="../QModules.qrc">
              <normaloff>:/QModules/icons/Merge-short_arrows_2.png</normaloff>:/QModules/icons/Merge-short_arrows_2.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QProgressBar" name="PBInputNegLoad">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>12</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="invertedAppearance">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="PBInputPosLoad">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>12</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>101</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="LInputOverload">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="LOscillatorFreq">
            <property name="text">
             <string>Oscillator Frequency</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="LEOscillatorFreq">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GBDemod">
       <property name="title">
        <string>Demodulation and Filtering</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="LPhase">
          <property name="text">
           <string>Phase</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="LTimeConstant">
          <property name="text">
           <string>Time Constant</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="LFilterOrder">
          <property name="text">
           <string>Filter Order</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="SBFilterOrder">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="SBTimeConstant">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> µs</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.001000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="SBPhase">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string>°</string>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="BAutoPhase">
          <property name="text">
           <string>Auto Phase</string>
          </property>
          <property name="icon">
           <iconset resource="../QModules.qrc">
            <normaloff>:/QModules/icons/Merge-short_arrows_2.png</normaloff>:/QModules/icons/Merge-short_arrows_2.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GBTrigger">
       <property name="title">
        <string>Trigger</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="LTriggerMode">
          <property name="text">
           <string>Mode</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="CBTriggerMode">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Continuous</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Extern Single</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="LTriggerEdge">
          <property name="text">
           <string>Edge</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="CBTriggerEdge">
          <item>
           <property name="text">
            <string>Rise</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fall</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="BForceTrigger">
          <property name="text">
           <string>Force</string>
          </property>
          <property name="icon">
           <iconset resource="../DynExpManager.qrc">
            <normaloff>:/DynExpManager/icons/Media-playback-start.svg</normaloff>:/DynExpManager/icons/Media-playback-start.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GBDigitization">
       <property name="title">
        <string>Signal Digitization</string>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="1" column="0">
         <widget class="QLabel" name="LSamplingRate">
          <property name="text">
           <string>Sampling Rate</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="LQuantity">
          <property name="text">
           <string>Quantity</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="CBQuantity">
          <item>
           <property name="text">
            <string>X</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Y</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>R</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>θ</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="LEnable">
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="CBEnable">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="SBSamplingRate">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> smpls/s</string>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="LProgress">
       <property name="text">
        <string>Acquisition progress</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="PProgress">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BPersist">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Persist parameters to instrument</string>
       </property>
       <property name="icon">
        <iconset resource="../DynExpManager.qrc">
         <normaloff>:/DynExpManager/icons/Document-save.svg</normaloff>:/DynExpManager/icons/Document-save.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SBRange</tabstop>
  <tabstop>BAutoRange</tabstop>
  <tabstop>LEOscillatorFreq</tabstop>
  <tabstop>SBPhase</tabstop>
  <tabstop>BAutoPhase</tabstop>
  <tabstop>SBTimeConstant</tabstop>
  <tabstop>SBFilterOrder</tabstop>
  <tabstop>CBTriggerMode</tabstop>
  <tabstop>CBTriggerEdge</tabstop>
  <tabstop>BForceTrigger</tabstop>
  <tabstop>CBQuantity</tabstop>
  <tabstop>SBSamplingRate</tabstop>
  <tabstop>CBEnable</tabstop>
  <tabstop>BPersist</tabstop>
 </tabstops>
 <resources>
  <include location="../DynExpManager.qrc"/>
  <include location="../QModules.qrc"/>
 </resources>
 <connections/>
</ui>
